lib/components/molecules/Quote/Quote.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './Quote.style';
+
+type Props = {
+  className: string,
+  content: string,
+  contentSrc: string,
+  author?: string,
+  company?: string,
+};
+
+const Quote = ({ content, contentSrc, author, company, className, ...otherProps }: Props): Node => {
+  return (
+    <div {...otherProps} className={classnames('quote-wrap', className)}>
+      <div className="quote__mark">&rdquo;</div>
```

Comment:
May be a flag here displayQuoteMarks as a boolean
or a prop named quoteIndicator which will default to "